Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+node('master') {
+
+  setBuildStatus('Building...', 'PENDING');
+
+  try {
+
+    stage('Checkout') {
+      checkout scm
+    }
+
+    stage('Build') {
+      sh '''#!/bin/bash -l
+        source ~/.bashrc
+        _utility/build_site.sh'''
+        setBuildStatus('Build Ok, deploying...', 'PENDING');
+    }
+
+    stage('Deploy') {
+      BRANCH_NAME=sh(script: 'git describe --all --exact-match 2>/dev/null | sed \'s=.*/==\'', returnStdout: true).trim()
+        AWS_REGION='eu-west-1'
+        WEBSITE_INDEX='index.html'
+        BUCKET_NAME=sh(script: "_utility/normalize_bucket_name.sh site-$BRANCH_NAME", returnStdout: true).trim()
+        sh "_utility/deploy_prb.sh $AWS_REGION $WEBSITE_INDEX $BUCKET_NAME"
+
+        BUCKET_URL="https://${BUCKET_NAME}.s3.amazonaws.com/$WEBSITE_INDEX"
+        SHORTEN_URL=sh(script: "_utility/shorten_url.sh $BUCKET_URL", returnStdout: true).trim()
+        setBuildStatus("Deployed: $SHORTEN_URL", 'SUCCESS');
+    }
+
+  } catch (e) {
+    setBuildStatus("Error while building", "UNSTABLE");
```

Comment:
Not sure about that, the limit for that string is 140 characters, And it will be displayed in the box that you can see at the screenshot, so it does not seem really usable.